test/sql/function/test_stringfunctions.cpp

```diff
@@ -473,11 +558,11 @@ TEST_CASE("BIT_LENGTH test", "[function]") {
 	result = con.Query("select BIT_LENGTH(NULL), BIT_LENGTH(''), BIT_LENGTH('\x24'), "
 	                   "BIT_LENGTH('\xC2\xA2'), BIT_LENGTH('\xE2\x82\xAC'), BIT_LENGTH('\xF0\x90\x8D\x88')");
 	REQUIRE(CHECK_COLUMN(result, 0, {Value()}));
-	REQUIRE(CHECK_COLUMN(result, 1, {0*8}));
-	REQUIRE(CHECK_COLUMN(result, 2, {1*8}));
-	REQUIRE(CHECK_COLUMN(result, 3, {2*8}));
-	REQUIRE(CHECK_COLUMN(result, 4, {3*8}));
-	REQUIRE(CHECK_COLUMN(result, 5, {4*8}));
+	REQUIRE(CHECK_COLUMN(result, 1, {0 * 8}));
```

Comment:
hah, these changes are made by `make format` 😂

Reply:
my `clang-format` version is 10.0:
```
➜ clang-format --version
clang-format version 10.0.0
```

Reply:
No problem